dev-client/src/components/projects/CreateProjectView/InnerForm.tsx

```diff
@@ -0,0 +1,82 @@
+import {HStack, Input, Text, VStack} from 'native-base';
```

Comment:
nit: this file looks like code from a file that got deleted from https://github.com/techmatters/terraso-mobile-client/pull/207

Reply:
Yes, I always miss this stuff :disappointed: